README.md

```diff
@@ -0,0 +1,44 @@
+skitter_api is a small wrapper around a set of [FlareSolverr](https://github.com/FlareSolverr/FlareSolverr) instances for load
+balancing and redundancy purposes. Essentially a series of docker containers
+comprised of gluetun providing a VPN connection and FlareSolverr providing CF
+workarounds are kept alive and proxied to.
+
+Currently these workers choose from a shared queue table stored in postgres;
+this is for ease-of-adapting from a previous project that was trivially
+shardable. This is far from ideal for a number of reasons. There's an initial
+effort to move to rabbitmq queues instead.
+
+Ensure you create a valid `etc/nginx/auth/skitter.htpasswd` file.
+
+`/etc/hosts` on the deploy target should contain an entry for polaris.fanfic.dev
+
+
```

Comment:
Kind of nitpicky, but does this extra blank line do anything in markdown? You removed some other instances of double blank lines so I'm not sure.

Reply:
Ah. 
> ...does this extra blank line do anything in markdown?

No. Just a blunder. Fixed it.